processing/data_collection/gazette/spiders/rj_sao_goncalo.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+from gazette.spiders.base import BaseGazetteSpider
+from gazette.items import Gazette
+
+
+class RjSaoGoncaloSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3304904"
+    DOWNLOAD_GAZETTE_URL = "https://servicos.pmsg.rj.gov.br/diario/{}.pdf"
+    start_date = datetime.date(1998, 2, 3)
+
+    name = "rj_sao_goncalo"
+    allowed_domains = ["servicos.pmsg.rj.gov.br"]
+    start_urls = ["https://servicos.pmsg.rj.gov.br/diario_oficial.php"]
+
+    def parse(self, response):
+        date = datetime.date.today()
+        start_date = self.start_date
+        while date >= start_date:
+            is_weekend = date.weekday() >= 5
+            if is_weekend:
+                date = date - datetime.timedelta(days=1)
+                continue
```

Comment:
I found this interesting, took it with a grain of salt at first but after testing this seems really nice :)

Reply:
Tested this from 2018 to now and as @rennerocha pointed out it does, in fact, not work because of older dates :/